fw_pipeline_io/fw_cli/commands.py

```diff
@@ -0,0 +1,137 @@
+"""
+Python functions for the Fllywheel CLI.
+"""
+import shutil
+from pathlib import Path
+import os
+import logging
+from uuid import uuid4
+
+log = logging.getLogger(__name__)
+
+
+class FWCLI:
+    def __init__(self, fw_cli_path: str = None):
+        """
+        Initialize the FWCLI object.
+
+        Parameters
+        ----------
+        fw_cli_path : str
+            Path to the Flywheel CLI.
+        """
+        if fw_cli_path is None:
+            fw_cli_path = shutil.which("fw")
+            if fw_cli_path is None:
+                raise ValueError(
+                    "Could not find Flywheel CLI. Please install it or pass a "
```

Comment:
It might make the error message too long, but it might be good for Metzger people who may not be as familiar with coding to add the pip installation command for Flywheel CLI as a hint.

Reply:
added helpful info for troubleshooting